lib/symgate/wordlist/client.rb

```diff
@@ -27,12 +27,12 @@ def enumerate_wordlists(context = [])
 
       # creates a wordlist with the specified name, context and scope (see auth:scope).
       # optionally, supply a list of entries to form the wordlist's initial content
-      def create_wordlist(name, context, entries = [])
+      def create_wordlist(name, context, readonly = false, entries = [])
         tries ||= 3 # TODO: Find out if we still need to do this!
```

Comment:
Someone should find this out.

Reply:
That was a thing because the symboliser sometimes used to return an error where it would create the wordlist OK but then fail to find it. See SOS-105 in the Widgit Jira. It seems that it may or may not have been fixed.